CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -1963,7 +1963,7 @@ int flake_encode_init()
 	struct FlakeEncodeParams
 	{
 		// compression quality
-		// set by user prior to calling flake_encode_init
+		// set by user prior to calling Flake_encode_init

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.